dotCMS/src/main/java/com/dotmarketing/util/UtilMethods.java

```diff
@@ -3680,4 +3680,21 @@ private static int getTrimRightPosition(final char[] value){
     public static <T> T isSetOrGet(final T toEvaluate, final T defaultValue){
         return UtilMethods.isSet(toEvaluate) ? toEvaluate : defaultValue;
     }
+
+
+    /**
+     * Finds if the length of the given value is valid
+     *
+     * @param value
+     * @param maxLength
+     * @param <T>
+     * @return
+     */
+    public static <T> boolean exceedsMaxLength(T value, int maxLength) {
+        if (value instanceof String) {
```

Comment:
you may want to user Charsequence instead of String to have more subclasses covered
set parameters to final too